app/controllers/recipie_controller.rb

```diff
@@ -1,9 +1,31 @@
 class RecipieController < ApplicationController
   before_action :authenticate_user!
   def show
-    @recipie = Recipie.find(params[:id])
+    @recipie = Recipie.find(params[:id])  
     @steps = @recipie.steps
+    @comments = []
+    @steps.each do |step|
+      @comments << step.comments.order("created_at")
+    end 
+    @commented_users = []
+    @comments.each do |comment|
+      comment.each do |com|
```

Comment:
I thought we will be deleting this right?

Reply:
Yeah ! but I was trying to get only those users who have commented . So the order of this will be according to the comments array . But I'm still not satisfied by the way I have done this , because I feel this is not the efficient way . It will be good if we can discuss about this  .